docs/components/toast/README.md

```diff
@@ -0,0 +1,248 @@
+                "description": "$emit event fired when opening the toast"
+              },
+              {
+                "name": "closed",
+                "arguments": "event",
+                "description": "$emit event fired when closing the toast"
+              }
+            ],
+          }
+        }
+      ]
+    }
+  ]
+}
+---
+
+<cdr-doc-table-of-contents-shell >
+
+# Overview
+
+CdrToast is a type of non-modal dialog used to communicate the status of a task or process. A proper toast is concise, time-relevant, and temporary.
+
+There are five different options for styling the toast, based on the [message type](../toast/#guidelines).
+
+## Default Toast with Icon Left
+
+A toast should be passed a contextually appropriate icon using the `icon-left` slot.
+
+<cdr-doc-example-code-pair repository-href="/src/components/CdrToast"
+:load-sprite="true"
+:sandbox-data="Object.assign({}, $page.frontmatter.sandboxData, {components: 'CdrToast, CdrButton, IconCheckFill'})" :model="{ opened: false }" >
+
+```html
+<div>
+  <cdr-button @click="opened = true">Trigger toast</cdr-button>
+  <!-- CdrToast should always be contained within an absolutely 
+  positioned element in the top right of your page -->
+  <div style="position: absolute; top: 2rem; right: 1.5rem">
+    <cdr-toast
+      :open="opened"
+      type="success"
+      @closed="opened = false">
+      <template #icon-left>
+        <icon-check-fill inherit-color>
+      </template>
+      I am a success toast
+    </cdr-toast>
+  </div>
+</div>
+```
+</cdr-doc-example-code-pair>
+
+## Persistent toast
+
+CdrToast provides an optional `autoDismiss` property to disable the auto-dismiss functionality.
+
+<cdr-doc-example-code-pair repository-href="/src/components/CdrToast"
+:load-sprite="true"
+:sandbox-data="Object.assign({}, $page.frontmatter.sandboxData, {components: 'CdrToast, CdrChip, IconWarningFill'})" :model="{ opened: false }" >
+
+```html
+<div>
+  <cdr-chip
+    :aria-pressed="opened"
+    @click="opened = true"
+  >
+    Trigger toast
+  </cdr-chip>
+  <!-- CdrToast should always be contained within an absolutely 
+  positioned element in the top right of your page -->
+  <div style="position: absolute; top: 2rem; right: 1.5rem">
+    <cdr-toast
+      :open="opened"
+      :auto-dismiss="false"
+      type="warning"
+      @closed="opened = false">
+      <template #icon-left>
+        <icon-warning-fill inherit-color>
+      </template>
+      I am a warning toast and must be dismissed by user action
+    </cdr-toast>
+  </div>
+</div>
+```
+</cdr-doc-example-code-pair>
+
+## Accessibility
+
+Many WCAG requirements are contextual to their implementation. To ensure that usage of this component complies with accessibility guidelines you are responsible for the following:
+- Add `role="alert"` for messages that are important or time sensitive to ensure that the messaging is immediately announced to screen readers. For other content, add `aria-live="polite"`.
```

Comment:
I think that this should use the requirements listed in transient status notifications , in general we would not want to use a toast as an alert so role="status" is more appropriate